scripts/tools/zap/zapfile_formatter.py

```diff
@@ -20,37 +20,83 @@
 from typing import List
 
 _DEFAULT_CLUSTER_REVISION_ATTRIBUTE = {
-    "name": "ClusterRevision",
-    "code": 0xFFFD,
-    "mfgCode": None,
-    "side": "server",
-    "type": "int16u",
-    "included": 1,
-    "storageOption": "RAM",
-    "singleton": 0,
-    "bounded": 0,
-    "defaultValue": "1",
-    "reportable": 1,
-    "minInterval": 1,
-    "maxInterval": 65534,
-    "reportableChange": 0
+    "entry": {
+        "name": "ClusterRevision",
+        "code": 0xFFFD,
+        "mfgCode": None,
+        "side": "server",
+        "type": "int16u",
+        "included": 1,
+        "storageOption": "RAM",
+        "singleton": 0,
+        "bounded": 0,
+        "defaultValue": "1",
+        "reportable": 1,
+        "minInterval": 1,
+        "maxInterval": 65534,
+        "reportableChange": 0
+    },
+    "clusterCode": None,
+    "clusterParamKey": "attributes",
+    "side": "server"
 }
 
 _DEFAULT_FEATURE_MAP_ATTRIBUTE = {
-    "name": "FeatureMap",
-    "code": 0xFFFC,
-    "mfgCode": None,
-    "side": "server",
-    "type": "bitmap32",
-    "included": 1,
-    "storageOption": "RAM",
-    "singleton": 0,
-    "bounded": 0,
-    "defaultValue": "0",
-    "reportable": 1,
-    "minInterval": 1,
-    "maxInterval": 65534,
-    "reportableChange": 0
+    "entry": {
+        "name": "FeatureMap",
+        "code": 0xFFFC,
+        "mfgCode": None,
+        "side": "server",
+        "type": "bitmap32",
+        "included": 1,
+        "storageOption": "RAM",
+        "singleton": 0,
+        "bounded": 0,
+        "defaultValue": "0",
+        "reportable": 1,
+        "minInterval": 1,
+        "maxInterval": 65534,
+        "reportableChange": 0
+    },
+    "clusterCode": None,
+    "clusterParamKey": "attributes",
+    "side": "server"
+}
+
+_TEST_EVENT_TRIGGERS_ENABLED_ATTRIBUTE = {
+    "entry": {
+        "name": "TestEventTriggersEnabled",
+        "code": 8,
+        "mfgCode": None,
+        "side": "server",
+        "type": "boolean",
+        "included": 1,
+        "storageOption": "External",
+        "singleton": 0,
+        "bounded": 0,
+        "defaultValue": "false",
+        "reportable": 1,
+        "minInterval": 1,
+        "maxInterval": 65534,
+        "reportableChange": 0
+    },
+    "clusterCode": 51,
+    "clusterParamKey": "attributes",
+    "side": "server"
+}
+
+_TEST_EVENT_TRIGGERS_CLIENT_COMMAND = {
+    "entry": {
+        "name": "TestEventTrigger",
+        "code": 0,
+        "mfgCode": None,
+        "source": "client",
+        "incoming": 1,
+        "outgoing": 0
+    },
+    "clusterCode": 51,
+    "clusterParamKey": "commands",
+    "side": "client"
```

Comment:
Why is this "side: client"?

Reply:
I misread some existing implementations, and definitely did the wrong thing here. Thanks for calling this out. I have fixed this in the script and updated the all-cluster and all-clusters-minimal.

Just regenerating right now, once I push new commits I will mark this as resolved